src/style.css

```
@@ -0,0 +1,44 @@
+
+body {
+  margin: inherit;
+}
+
+.stars {
+  display: flex;
+  flex-direction: column;
+  width: 100px;
+}
+
+.star {
+  background-image: url(./images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  height: 16px;
+  width: 16px;
+  margin-right: 4px;
+}
+
+.stars div:nth-child(n) {
+  display: flex;
+  justify-content: space-between;
+}
+
+.stars--1 div:nth-child(1) {
+  background-image: url(./images/star-active.svg);
+}
+
+.stars--2 div:nth-child(-n+2) {
+  background-image: url(./images/star-active.svg);
+}
+
+.stars--3 div:nth-child(-n+3) {
+  background-image: url(./images/star-active.svg);
+}
+
+.stars--4 div:nth-child(-n+4) {
+  background-image: url(./images/star-active.svg);
+}
+
+.stars--5 div:nth-child(n) {
+  background-image: url(./images/star-active.svg);
+}
```

```suggestion
.stars--1 .star:nth-child(1),
.stars--2 .star:nth-child(-n+2),
.stars--3 .star:nth-child(-n+3),
.stars--4 .star:nth-child(-n+4),
.stars--5 .star:nth-child(n) {
  background-image: url(./images/star-active.svg);
}
```
давай будемо використовувати класи як селектори, а не теги,
також ці селектори варто скомбінувати